### PR TITLE
Git repository size and contributors list

### LINKED ISSUE #378: Script to find size of Github Repository + List of Contributors
# Description

Create a Script to find the size of Github Repository + List of Contributors 

## Language

- [X] Python
- [X] Javascript

## Type of issue

_Please delete options that are not relevant._

- [X] Feature

### PR DESCRIPTION
**Pull request by kaustubhgupta**
# Description

This script will return a CSV file containing the Size of the repository and details of the contributors of the repo.

Fixes #378 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Git_repo_size_contributors/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+`python app.py username_of_repo name_of_repo personal_access_token`
+
+
+Before running the script, make sure you have the required dependicies. Not sure about them? Run the following command before running the actual script:
+
+`pip install -r reqirements.txt`
+
+# Working Demo
+![](preview.PNG)
```
Comment (HarshCasper) [inline Python/Git_repo_size_contributors/README.md:14]: It should be image

File: Python/Git_repo_size_contributors/app.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    size_request = requests.get(size_link)
+    data_received = json.loads(size_request.content)
+    size = data_received["size"]/1024
+
+    data = {}
+    data['Size of Repo (MB)'] = size
+
+    page_no = 1
+    progress_bar = tqdm(unit=' Page')
+    while True:
```
Comment (HarshCasper) [inline Python/Git_repo_size_contributors/app.py:21]: Add better comments to the Code

File: Python/Git_repo_size_contributors/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+requests
```
Comment (HarshCasper) [inline Python/Git_repo_size_contributors/requirements.txt:1]: Add versions for your requirements as well

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Kindly make the necessary changes

**Comment by kaustubhgupta** [comment]
@HarshCasper all changes done!

**Pull request merged**